vss-extension.json

```
@@ -0,0 +1,93 @@
+{
+  "manifestVersion": 1,
+  "id": "infracost-tasks",
+  "name": "Infracost Tasks",
+  "version": "0.0.1",
+  "publisher": "Infracost",
+  "targets": [
+    {
+      "id": "Microsoft.VisualStudio.Services"
+    }
+  ],
+  "description": "Tasks for setting up Infracost in your Azure DevOps pipeline and showing cloud cost estimates for Terraform in pull requests.",
```

Seems like this is a proper name for the service, need to stick with it everywhere.
```suggestion
  "description": "Tasks for setting up Infracost in your Azure Pipelines and showing cloud cost estimates for Terraform in pull requests.",
```

---

👍  lgtm